packages/shared/styles/components/atoms/SfQuantitySelector.scss

```
@@ -1,22 +1,71 @@
 @import "../../helpers";
 
 .sf-quantity-selector {
-  --input-width: 1.875rem;
-  --input-height: 1.6875rem;
-  input {
-    --input-padding: 0;
-    --input-margin: 0;
-    --input-border-width: 1px;
-    text-align: center;
-    transition: border-color 150ms linear;
-    &:focus {
-      --input-border: 1px solid var(--c-primary);
-      --input-border-width: 1px 1px 0 1px;
-      --input-padding: 0 0 1px 0;
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: center;
+  height: var(--quantity-selector-height, 3.5rem);
+  width: var(--quantity-selector-width, 23.4375rem);
+  padding: var(--quantity-selector-padding, 0);
+  background: var(--quantity-selector-background);
+  &__button {
+    --button-color: var(--c-text);
+    --button-background: var(--c-light);
+    --button-padding: 0;
+    --button-size: 1.625rem;
+    border-radius: var(--quantity-selector-button-border-radius, 50%);
+    &:hover {
+      --button-background: var(--c-light);
+      box-shadow: none;
     }
   }
+  &__input {
+    --input-margin: 0;
+    --input-padding: 0;
+    --input-border: none;
+    --input-height: 3.5rem;
+    --input-text-align: center;
+    --input-bar-display: none;
+  }
+  &::before {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 1px;
+    background: var(--c-light);
+    transition: none;
+    content: "";
+  }
   @include for-desktop {
-    --input-width: 3.125rem;
-    --input-background: var(--c-light);
+    --quantity-selector-background: var(--c-light);
+    --quantity-selector-width: 6.75rem;
+    --quantity-selector-height: 3.25rem;
+    --quantity-selector-padding: 0 0.625rem;
+    --quantity-selector-button-border-radius: 0;
+    --quantity-selector-font-size: 22px;
```

```suggestion
    --quantity-selector-font-size: 22px;
```
--font-xl //24px it shoul be ok